src/containers/Tenant/utils/schemaActions.ts

```diff
@@ -1,15 +1,16 @@
 import {Dispatch} from 'react';
+import copy from 'copy-to-clipboard';
```

Comment:
Why did you decide not to use `CopyToClipboard` from `uikit`?

Reply:
Because there is a component in `uikit`, but in this case I needed only `copy` action to add it to an option in schema tree